### PR TITLE
PR for revamped settings app

### PR DESCRIPTION
**Pull request by Nutlope**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature

**What is the new behavior (if this is a feature change)?**
- adding sidebarName prop for new settings side menu
- Rewrote certain names and descriptions

### CODE REVIEW

File: tools/all-settings.json [hunk 1]
```diff
@@ -1,169 +1,235 @@
...
+    },
+    {
+      "settingName": "pty.path",
+      "sidebarName": "PTY Path",
+      "title": "Set $PATH of Fig's Pseudoterminal",
+      "description": "Sets the PATH variable used in pseudoterminals.",
+      "type": "text",
+      "default": "",
+      "details": "The error text 'commands not found' often appears if this path is not set correctly.",
+      "sidebarHeading": "Psuedoterminal",
```
Comment (QuiiBz) [inline tools/all-settings.json:167]:
```suggestion
      "sidebarHeading": "Pseudoterminal",
```

File: tools/all-settings.json [hunk 2]
```diff
@@ -1,169 +1,235 @@
...
+    },
+    {
+      "settingName": "pty.rc",
+      "sidebarName": "Init File for",
+      "title": "Init File for ",
+      "description": "A file that will be sourced when Fig creates a pseudoterminal.",
+      "type": "text",
+      "default" : "~/.fig/user/ptyrc",
+      "details" : "Add environment variables to this file, in order for commands like `aws` to use them when generating completions.",
+      "sidebarHeading": "Psuedoterminal"
```
Comment (QuiiBz) [inline tools/all-settings.json:178]:
```suggestion
      "sidebarHeading": "Pseudoterminal"
```

**Review by QuiiBz — COMMENTED** [review]
Looks like the whole file has two extra spaces at the beginning of each line

**Review by QuiiBz — APPROVED** [review]

**Review by mschrage — CHANGES_REQUESTED** [review]
Just had a look at this. I think that `sidebarName` is does not need to exist. We should just put this information in the `title` property. (I like your updates to the copy, just don't think we need a new property.)

I also would make `sidebarHeading` more generic. Perhaps `category` instead?

**Review by mschrage — APPROVED** [review]
🎉

**Pull request merged**